eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -66,6 +66,9 @@ jobs:
         displayName: "Use Python 3.6"
         inputs:
           versionSpec: "3.6"
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
```

Comment:
Lets use absolute paths `/eng/common/pipelines/templates/steps/verify-links.yml` here and in the aggregate reports.